Commands_Functions/Icommand.py

```diff
@@ -0,0 +1,18 @@
+from abc import ABC, abstractmethod
```

Comment:
This should be Abstract command not interface anymore

Reply:
we changed it to interface as there is no function implemented on this class